ah, one of the APIs changed, sorry:

```
src/afl-tmin.c:175:6: error: conflicting types for ‘update_bitmap_score’; have ‘void(afl_state_t *, struct queue_entry *)’ {aka ‘void(struct afl_state *, struct queue_entry *)’}
  175 | void update_bitmap_score(afl_state_t *afl, struct queue_entry *q) {
      |      ^~~~~~~~~~~~~~~~~~~
In file included from src/afl-tmin.c:40:
include/afl-fuzz.h:1193:6: note: previous declaration of ‘update_bitmap_score’ with type ‘void(afl_state_t *, struct queue_entry *, _Bool)’ {aka ‘void(struct afl_state *, struct queue_entry *, _Bool)’}
 1193 | void update_bitmap_score(afl_state_t *, struct queue_entry *, bool);
      |      ^~~~~~~~~~~~~~~~~~~
make: *** [GNUmakefile:500: afl-tmin] Error 1
```
has another bool var now

--- a/src/afl-tmin.c
+++ b/src/afl-tmin.c
@@ -172,9 +172,10 @@ void show_stats(afl_state_t *afl) {
   (void)afl;
 }
 
-void update_bitmap_score(afl_state_t *afl, struct queue_entry *q) {
+void update_bitmap_score(afl_state_t *afl, struct queue_entry *q, bool add_to_queue) {
   (void)afl;
   (void)q;
+  (void)add_to_queue;
 }
 
 fsrv_run_result_t fuzz_run_target(afl_state_t *afl, afl_forkserver_t *fsrv, u32 i) {

